src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+  font-weight: 600;
+}
+
+.search-bar__label,
+.small-lab {
+  display: block;
+  position: relative;
+  margin-bottom: 20px;
+}
+
+.search-bar__label::before {
+  content: "";
+  display: block;
+  background: url(images/Search.svg) no-repeat;
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  height: 19px;
+  width: 19px;
+  background-size: contain;
+}
+
+.small-lab::before {
+  content: "";
+  position: absolute;
+  display: block;
+  background: url(images/Search.svg) no-repeat;
```

Comment:
you repeat these styles few times
```
  content: "";
  position: absolute;
  display: block;
  background: url(images/Search.svg) no-repeat;
  background-size: contain;
```
create a common class for them or something else